openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -874,6 +875,111 @@ def build_lipsync(project_name: str, shot_name: str):
             )
 
 
+def build_fabrication(project_name, asset_name):
+    """Build fabrication workfile
+
+    Args:
+        project_name (str): The current project name from OpenPype Session.
+        asset_name (str): The current asset name from OpenPype Session.
+    """
+
+    # Get subsets
+    asset_doc = get_asset_by_name(
+        project_name, "LightSetupBank", fields=["_id"]
+    )
+    subsets = get_subsets(
+        project_name,
+        asset_ids={asset_doc["_id"]},
+        fields=["_id", "name", "data"],
+    )
+
+    representations = []
+    # Match and download and load subset
+    for subset in subsets:
+        raw_asset_name = asset_name.split("_")[0].lower()
+        raw_subset_name = (
+            subset["name"].strip("lighting").rsplit("_", 1)[0].lower()
+        )
+        if raw_asset_name == raw_subset_name:
+            representation = download_subset(
+                project_name,
+                "LightSetupBank",
+                subset["name"],
+            )
+            representations.append(representation)
+            load_subset(
+                project_name, representation, "AppendBlenderLightingLoader"
+            )
+    # Wait for downloads to be finished
+    wait_for_download(project_name, representations)
+
+    # Create setdress instance
+    bpy.ops.scene.create_openpype_instance(
+        creator_name="CreateWoollySetdress",
+        asset_name=asset_name,
+        subset_name="setdressMain",
+        gather_into_collection=True,
+    )
```

Comment:
```suggestion
    bpy.ops.scene.create_openpype_instance(
        creator_name="CreateWoollySetdress",
        asset_name=asset_name,
        subset_name="setdressMain",
        gather_into_collection=True,
        use_selection=False,
    )
```
Since you don't need to use current selection, might as well set `use_selection` to `False`.

Reply:
use_selection is `False` by default